### PR TITLE
Init workflow

### PR DESCRIPTION
**Pull request by matthewwong525**

### CODE REVIEW

**Comment by matthewwong525** [comment]
retest this please

**Comment by matthewwong525** [comment]
retest this please

**Comment by matthewwong525** [comment]
retest this please

File: sniffles.wdl [hunk 1]
```diff
@@ -0,0 +1,67 @@
+version 1.0
+
+workflow sniffles {
+    input {
+        File sortedBamFile
+        String outputFileNamePrefix
+    }
+    parameter_meta {
+        sortedBamFile: "Input directory (directory of the nanopore run)"
```
Comment (mlaszloffy) [inline sniffles.wdl]: Can the description be updated to "Input bam file (must be sorted)"

File: sniffles.wdl [hunk 2]
```diff
@@ -0,0 +1,67 @@
+version 1.0
+
+workflow sniffles {
+    input {
+        File sortedBamFile
+        String outputFileNamePrefix
```
Comment (mlaszloffy) [inline sniffles.wdl]: Maybe a default of `basename(sortedBamFile, ".bam")`

**Review by mlaszloffy — CHANGES_REQUESTED** [review]

**Comment by matthewwong525** [comment]
retest this please

**Review by mlaszloffy — APPROVED** [review]

File: sniffles.wdl [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    parameter_meta {
+        sortedBamFile: "Input bam file (must be sorted)"
+        outputFileNamePrefix: "Variable used to set the outputfile name"
+        additionalParameters: "Additional parameters to be added to the sniffles command"
+    }
+
+    meta {
+        author: "Matthew Wong"
+        email: "m2wong@oicr.on.ca"
```
Comment (lheisler) [inline sniffles.wdl]: this email address will disappear once you leave OICR.  absolutely fine to include your name, but i don't think it makes sense to use this address.

File: sniffles.wdl [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        description: "Workflow to run Structural Variant caller"
```
Comment (lheisler) [inline sniffles.wdl]: description can be more specific, "Workflow to run the Structural Variant caller Sniffles on Oxford Nanopore Data"

**Review by lheisler — APPROVED** [review]

**Pull request merged**